spec/factories/species.rb

```diff
@@ -0,0 +1,20 @@
+FactoryGirl.define do
+  factory :species do
+    name {FFaker::Lorem.paragraph}
+    description {FFaker::Lorem.paragraph}
+    source_page {(rand * 100).to_i}
+    source_book {"Force and Destiny Core Rulebook"}
+    brawn {1}
```

Comment:
`source_page {(rand * 100).to_i}`
After removing the unnecessary {}'s this line throws "Undefined * method", can I leave that line with curly braces or is there another way to do it?